yksgi/src/lib.rs

```diff
@@ -13,9 +13,10 @@ use std::slice;
 mod llvmbridge;
 use llvmbridge::{get_aot_original, BasicBlock, Module, Type, Value};
 
-static ALWAYS_SKIP_FUNCS: [&str; 5] = [
+static ALWAYS_SKIP_FUNCS: [&str; 6] = [
     "llvm.lifetime",
     "llvm.lifetime.end.p0i8",
+    "llvm.lifetime.end.p0",
```

Comment:
I'm uncertain if this is the correct way to do this. I suspect the `.p0` and `.p0i8` suffixes are arbitrary and are part of a mangling scheme.

Should we have another `ALWAYS_SKIP_FUNCS_WITH_PREFIX` list which is contains (for now) the prefix `llvm.lifetime` ?

Reply:
Since this is in `yksgi`, that is going to be removed (right?) maybe we can ignore this.

Reply:
This is all gonna go away when we switch to stack recovery, so I'm not too fussed that this isn't the best way to handle these cases.

Reply:
I tend to agree.